src/wp-admin/includes/class-wp-comments-list-table.php

```diff
@@ -327,32 +345,36 @@ protected function extra_tablenav( $which ) {
 		if ( ! isset( $has_items ) ) {
 			$has_items = $this->has_items();
 		}
-?>
+		?>
 		<div class="alignleft actions">
-<?php
+		<?php
 		if ( 'top' === $which ) {
-?>
+			?>
 			<label class="screen-reader-text" for="filter-by-comment-type"><?php _e( 'Filter by comment type' ); ?></label>
 			<select id="filter-by-comment-type" name="comment_type">
 				<option value=""><?php _e( 'All comment types' ); ?></option>
-<?php
+			<?php
 				/**
 				 * Filters the comment types dropdown menu.
 				 *
 				 * @since WP-2.7.0
 				 *
 				 * @param array $comment_types An array of comment types. Accepts 'Comments', 'Pings'.
 				 */
-				$comment_types = apply_filters( 'admin_comment_types_dropdown', array(
-					'comment' => __( 'Comments' ),
-					'pings' => __( 'Pings' ),
-				) );
+				$comment_types = apply_filters(
+					'admin_comment_types_dropdown',
+					array(
+						'comment' => __( 'Comments' ),
+						'pings'   => __( 'Pings' ),
+					)
+				);
 
-				foreach ( $comment_types as $type => $label )
-					echo "\t" . '<option value="' . esc_attr( $type ) . '"' . selected( $comment_type, $type, false ) . ">$label</option>\n";
+			foreach ( $comment_types as $type => $label ) {
```

Comment:
indentation?

Reply:
Seems like the lines above are indented too far.